src/components/emby-slider/emby-slider.css

```diff
@@ -87,6 +87,10 @@ _:-ms-input-placeholder {
     transform: scale(1.6);
 }
 
+.mdl-slider:focus::-webkit-slider-thumb {
+    transform: scale(1.6);
+}
+
```

Comment:
Won't this have the same focus issue we are trying to solve in your scaling pull request? We only want focus to show for TV layouts normally.

Reply:
It is also scaled on hover.
https://github.com/jellyfin/jellyfin-web/blob/8aae12d0147e7a421d41b40a7cb75f9ca3d2d249/src/components/emby-slider/emby-slider.css#L86-L88

But it is not a problem to add condition in `emby-slider`.

Just got a thought. In fact, the focus should be displayed if keyboard navigation is enabled. And keyboard navigation should (?) be enabled only on TV. But there is also TAB navigation, so maybe I'm wrong.